sound/soc/sof/Kconfig

```diff
@@ -117,12 +117,12 @@ config SND_SOC_SOF_DEBUG_VERBOSE_IPC
 	  if you are trying to debug IPC with the DSP firmware.
           If unsure select "N".
 
-config SND_SOC_SOF_DEBUG_FORCE_IPC_POSITION
-	bool "SOF force to use IPC for position update on SKL+"
+config SND_SOC_SOF_DEBUG_DISABLE_NOWAKEUP
+	bool "SOF to disable non-interrupt mode for position update on SKL+"
 	help
-	  This option force to handle stream position update IPCs and run pcm
-	  elapse to inform ALSA about that, on platforms (e.g. Intel SKL+) that
-	  with other approach (e.g. HDAC DPIB/posbuf) to elapse PCM.
+	  This option forces use of pcm position update from IPCs. IPC updates
+	  are used even on platforms where other mechanisms,
+	  (e.g. HDAC DPIB/posbuf), are available to query hardware position.
```

Comment:
no. this option forces the use of period interrupts, that's not correlated with IPC updates.

Reply:
My bad, sorry @fredoh9 , I proposed the above wording in an earlier comment.

But yeah, important to align commit+kconfig+code -- I also got confused that this is about forcing the mechanism used to query the position, but indeed that's not the case and code part seems clear to me now.

FORCE_PERIOD_WAKEUPS would avoid the double-negative description, but I think DISABLE_WAKEUP in current patch is ok and that can be understood in context of the SNDRV_PCM_INFO_NO_PERIOD_WAKEUP API.